src/main/java/net/revelc/code/impsort/ImpSort.java

```diff
@@ -131,8 +134,11 @@ public Result parseFile(final Path path, final byte[] buf) throws IOException {
       parseResult.getProblems().forEach(System.out::println);
       return new ImpSortException(path, Reason.UNABLE_TO_PARSE);
     });
-    if (!parseResult.isSuccessful()) {
-      parseResult.getProblems().forEach(System.out::println);
+    List<Problem> relevantProblems = ignoreIrrelevantParseErrors
```

Comment:
I think reportableProblems, or just problems, is better. Relevancy isn't quite the term I'd want to use.